Parsers/ASimRegistryEvent/Parsers/vimRegistryEventMicrosoftSecurityEvent.yaml

```diff
@@ -12,7 +12,8 @@ References:
   Link: https://aka.ms/ASimRegistryEventDoc
 - Title: ASIM
   Link: https:/aka.ms/AboutASIM
-Description: ASIM Security Events Registry Creation Event Parser (event number 4657)
+Description: |
+  This ASIM parser supports normalizing Microsoft Security Event logs, produced by the Microsoft AMA connector, to the ASIM Registry Event normalized schema (event number 4657).
```

Comment:
-> "This ASIM parser supports normalizing Microsoft Security Events (event number 4657), produced by the Microsoft AMA connector, to the ASIM Registry Event normalized schema."

Changes (which will repeat below):
- Moved (...) earlier.
- Event logs -> Events
- Dot at the end